pkg/distro/BUILD

```diff
@@ -19,7 +19,7 @@ pkg_tar(
     mode = "0444",
     # Make it owned by root so it does not have the uid of the CI robot.
     owner = "0.0",
-    package_dir = ".",
+    package_dir = "pkg",
```

Comment:
This changes the location of the rules. Everyone using it would have to change from @rules_pkg//:pkg.bzl to @rules_pkg//pkg:pkg.bzl

Reply:
Yes, that was the intent of the original issue.

Even if this code is not needed, the rest of it is to ensure that the experimental RPM generator lands in the release tarball.  Let me clean this up.

Reply:
Hmm... I've been thinking yesterday that this is something we might want to do anyway.
How about we discuss an optimal arrangement for the code first, and then move to that.

Reply:
That is,  move most of the files from pkg/*.bzl to pkg/pkg/*.bzl, but leave distro and tests at the same level.  Now the distribution tarball is only  pkg/... experimental/... and the top level BUILD and license.  We can put in a forwarder for a while so code using pkg:pkg.bzl will still work during migration.

WDYT?

Reply:
@dannysullivan because that move would impact docker's near term plan to point at archive.py

Reply:
> That is, move most of the files from pkg/.bzl to pkg/pkg/.bzl, but leave distro and tests at the same level. Now the distribution tarball is only pkg/... experimental/... and the top level BUILD and license. We can put in a forwarder for a while so code using pkg:pkg.bzl will still work during migration.
>
> WDYT?

Makes sense to me.  We may even additionally want to move to the model of:

- `pkg/defs.bzl` provides forwarders for all non-experimental rules
- `pkg/internal/*.bzl` provides rule implementations

Not sure where the python and other scripts would go.  Same directory?  experimental/ could follow the same pattern.

In the meantime, I removed the line in question and included all the files needed to run the experimental rpm packager.

Reply:
I'm not a fan of internal. It makes the code harder to trace and, without enforced visibility, it does not ensure that people won't depend on it anyway.